### PR TITLE
Test foundry

### PR DESCRIPTION
**Pull request by 0xenj**
problems

### CODE REVIEW

File: contracts/test/Test-enzo.t.sol [hunk 1]
```diff
@@ -3,13 +3,59 @@ pragma solidity ^0.8.19;
...
+    notOwner4 = vm.addr(9);
+    notOwner5 = vm.addr(10);
+
+    vm.roll(block.number + 1);
+    vm.warp(block.timestamp + 100);
+  }
+
+  function test_owner_and_threshold_set() public {
+    vm.prank(owner1);
+    multiSignature = new SimpleMultiSignature([owner1, owner2, owner3, owner4, owner5], 3);
```
Comment (marc-aurele-besner) [inline contracts/test/Test-enzo.t.sol:49]: by doing 
```solidity
[owner1, owner2, owner3, owner4, owner5]
```
you are effectively passing a address[5] to the constructor, while the contract expect a address[]
so you need to first define in memory or in storage of the test a unfix array of address and push the owners address into it, then use that variable

**Review by marc-aurele-besner — CHANGES_REQUESTED** [review]

File: .vscode/settings.json [hunk 2]
```diff
@@ -9,4 +9,3 @@
     "editor.tabSize": 2,
```
Comment (Guillaume-web) [inline .vscode/settings.json:9]: i don't want your setting.json vscode !! X)

**Review by Guillaume-web — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (0xenj) [reply .vscode/settings.json:9]: j'ai rien changé pourtant

Comment (Guillaume-web) [reply .vscode/settings.json:9]: Non rien n'a changé, c'est juste qu'il y avait un saut de ligne que tu as supprimé

**Review by marc-aurele-besner — CHANGES_REQUESTED** [review]
your pr is out of date with the main branch, but also in the last PR that was merge, I had made a comment to be fixed, a semi colon missing on line 39 of the contract, and the PR got merged without the fix, so please upgrade your branch and fix this issue, both check need to pass before this PR get merged

**Review by marc-aurele-besner — APPROVED** [review]

**Review by Guillaume-web — APPROVED** [review]

**Pull request merged**